Cargo.toml

```diff
@@ -41,7 +41,6 @@ googleapis-raw = { version = "0", path = "vendor/mozilla-rust-sdk/googleapis-raw
 # syncserver to either fail to either compile, or start. In those cases, try
 # `cargo build --features grpcio/openssl ...`
 grpcio = { version = "0.8" }
-hyper = "0.13.10"  # RUSTSEC-2021-0020 requires 0.13.10+ or 0.14.3+
```

Comment:
Note if we ever needed to bump this again (for say a RUSTSEC) we don't need an entry for it in Cargo.toml, we can usually just do a `cargo update -p hyper`